uktrade/data-hub-frontend
# Persist my investment projects filters with sessionStorage

## Description of change

Persists the filters and sort on the personalised homepage so that users do not have to set them again when they navigate elsewhere and come back.

## Test instructions

Go to personalised homepage and select some filters / sort - navigate to other pages and come back and the filters should stay applied.

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
